packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -1876,6 +1876,91 @@ account_pref:
     expires: never
     data_sensitivity:
       - interaction
+  google_unlink_submit:
+    type: event
+    description: |
+      The user clicked "Unlink" from account preferences for their Google linked account.
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-10050
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1844121
+    expires: never
+    data_sensitivity:
+      - interaction
+    extra_keys:
+      reason:
+        description: If reason is "create_password", the user must set a password before they can confirm the account unlinking.
+        type: string
+  google_unlink_submit_confirm:
+    type: event
+    description: |
+      The user clicked "Unlink" from the confirmation modal for their Apple linked account.
```

Comment:
It says "Apple" here but the name of the event starts with "google"

Reply:
Ah, thank you, a missed copy & paste.